### ISSUE TITLE
Trigger warnings

### ISSUE THREAD

**Issue by zpalmtree** [amwmedia/eslint-plugin-woke#2]
hello sir

please add trigger warnings to your files. I opened the file `racism.js` and some of the language there really hurt me, as a PoC I am not sure if we should allow words like this on github, maybe we could have an external word list stored somewhere so fellow allies do not have to view these slurs while still making sure their codebase is free of niggas

**Comment by amwmedia** [comment]
Absolutely! Maybe a comment at the top of each of the category files? Would you mind putting together a PR?

**Comment by andrienko** [comment]
Or maybe could just base64encode them, so they're not easily readable here on github. Could encode them separately, to maintain git history when words added separately - and decode before use (not on the go, cuz that will eat additional millisecond or two, which is important when linting)
